frontend/src/karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       reports: ['html', 'lcovonly'],
       fixWebpackSourcePaths: true
     },
-    reporters: ['progress', 'kjhtml'],
+    //reporters: ['progress', 'kjhtml'],
```

Comment:
could you please comment 11th line of the file as well?https://github.com/andreytlt13/FaceAnalytics/pull/27/files#diff-d9faf0df5cc03bac2efa452be0e266d2R11

Reply:
> could you please comment 11th line of the file as well?https://github.com/andreytlt13/FaceAnalytics/pull/27/files#diff-d9faf0df5cc03bac2efa452be0e266d2R11

thanks for finding that, it's done